spiketools/utils/trials.py

```diff
@@ -83,3 +85,32 @@ def split_trials_by_condition_array(trials, conditions):
         out[condition] = trials[np.where(np.array(conditions)==condition)[0], :]
 
     return out
+
+
+def recombine_trial_data(times_trials, values_trials, axis=None):
+    """Recombine data across trials.
+
+    Parameters
+    ----------
+    times_trials : list of 1d array
+        Epoched timestamps.
+    values_trials : list of 1d or 2d array
+        Epoched trial data.
+    axis : {-1, 0, 1}, optional
```

Comment:
What does `-1` mean here?

Reply:
Good question! I had to go double check lol. 
-1 is used if the the input arrays are specified to be 1d vectors, which don't have an orientation (as defined in numpy). This should be made explicit here, and across the any other mentions of the axis argument.

Reply:
Wait no, this isn't even correct lol. vector is also 0. -1 is a default answer that indicates that the axis could not be inferred (for example, if given an empty axis). I'm checking through to fix the docs on this now.